tasks/util/browserify_wrapper.js

```diff
@@ -1,14 +1,10 @@
-var fs = require('fs');
 var path = require('path');
 
-var browserify = require('browserify');
-var minify = require('minify-stream');
-var derequire = require('derequire');
-var through = require('through2');
+var webpack = require('webpack');
 
-var strictD3 = require('./strict_d3');
+var config = require('../../webpack.config.js');
 
-/** Convenience browserify wrapper
+/** Convenience bundle wrapper
```

Comment:
might want to change the filename too 😄

Reply:
I would definitely change that in a separate PR to make it easier to track the changes here.